yrs/src/transaction.rs

```diff
@@ -92,109 +44,120 @@ impl Transaction {
 
     /// Encodes the difference between remove peer state given its `state_vector` and the state
     /// of a current local peer
-    pub fn encode_diff<E: Encoder>(&self, state_vector: &StateVector, encoder: &mut E) {
+    fn encode_diff<E: Encoder>(&self, state_vector: &StateVector, encoder: &mut E) {
         self.store().encode_diff(state_vector, encoder)
     }
 
-    pub fn encode_diff_v1(&self, state_vector: &StateVector) -> Vec<u8> {
+    fn encode_diff_v1(&self, state_vector: &StateVector) -> Vec<u8> {
         let mut encoder = EncoderV1::new();
         self.encode_diff(state_vector, &mut encoder);
         encoder.to_vec()
     }
 
-    /// Returns a [Text] data structure stored under a given `name`. Text structures are used for
-    /// collaborative text editing: they expose operations to append and remove chunks of text,
-    /// which are free to execute concurrently by multiple peers over remote boundaries.
-    ///
-    /// If not structure under defined `name` existed before, it will be created and returned
-    /// instead.
-    ///
-    /// If a structure under defined `name` already existed, but its type was different it will be
-    /// reinterpreted as a text (in such case a sequence component of complex data type will be
-    /// interpreted as a list of text chunks).
-    pub fn get_text(&mut self, name: &str) -> Text {
-        let mut c = self
-            .store_mut()
-            .get_or_create_type(name, None, TYPE_REFS_TEXT);
-        c.store = Some(self.store.clone());
-        Text::from(c)
+    fn encode_state_as_update<E: Encoder>(&self, sv: &StateVector, encoder: &mut E) {
```

Comment:
These methods are not defined straight on `ReadTxn` which is a trait, so `pub` encapsulation modifier is no longer needed.